website/src/pages/v1/serve/index.mdx

```diff
@@ -44,6 +55,21 @@ mesh-serve --supergraph=supergraph.graphql
 **Note:** If you use GraphQL Hive or Apollo GraphOS, you can also configure Mesh Serve to poll the
 supergraph from the corresponding schema registry.
 
+### Option 2: Running with docker
```

Comment:
I think this option should be the first one.

Reply:
Sure, updated